tensorflow_model_optimization/python/core/clustering/keras/clustering_registry.py

```diff
@@ -127,14 +162,33 @@ def get_clusterable_weights():
               for weight_name in cls._weight_names(layer)]
 
     def get_clusterable_weights_rnn():  # pylint: disable=missing-docstring
-      if isinstance(layer.cell, clusterable_layer.ClusterableLayer):
-        raise ValueError(
-            'ClusterableLayer is not yet supported for RNNs based layer.')
-
-      clusterable_weights = [
-          ('kernel', layer.cell.kernel),
-          ('recurrent_kernel', layer.cell.recurrent_kernel),
-      ]
+      def get_clusterable_weights_rnn_cell(cell, i):
+        # Cell weights will be a list of tuples in RNN or
+        # when are wrapped by the StackedRNNCell layer
+        # The weight names will have indices attached only
+        # for the registry
+        if cell.__class__ in cls._SUPPORTED_RNN_CELLS:
+          cell_weights = []
+          cell_weights.append(('kernel/' + str(i), cell.kernel))
+          cell_weights.append(('recurrent_kernel/' + str(i),
+                               cell.recurrent_kernel))
+          return cell_weights
+
```

Comment:
Nit: would this not be more clear with 
```
return [('kernel/' + str(i), cell.kernel),  ('recurrent_kernel/' + stri(i), cell.recurrent_kernel)]
``` 
As previously?

Reply:
Thanks, done.